openwisp_notifications/migrations/0001_initial.py

```diff
@@ -0,0 +1,10 @@
+# Minimal stub migration for openwisp_notifications
```

Comment:
Adding `openwisp_notifications` here is most likely wrong, what is prompting you to do this?